### PR TITLE
Run tests for missing python versions

### PR DESCRIPTION
**Pull request by bmillemathias**
##### SUMMARY
Testing build on python 2.6

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME
tests

### CODE REVIEW

File: shippable.yml [hunk 1]
```diff
@@ -34,11 +35,13 @@ matrix:
     - env: T=devel/linux/ubuntu1604/1
     - env: T=devel/linux/ubuntu1804/1
 
+    - env: T=devel/cloud/2.6/1
     - env: T=devel/cloud/2.7/1
     - env: T=devel/cloud/3.6/1
 
     # For Ansible 2.10, use a combination of different targets
     - env: T=2.10/sanity/1
+    - env: T=2.10/units/2.6/1
```
Comment (felixfontein) [inline shippable.yml:44]:
```suggestion
```
We only run limited tests with stable-2.10. (Same for 2.9.)

File: shippable.yml [hunk 2]
```diff
@@ -34,11 +35,13 @@ matrix:
     - env: T=devel/linux/ubuntu1604/1
     - env: T=devel/linux/ubuntu1804/1
 
+    - env: T=devel/cloud/2.6/1
```
Comment (felixfontein) [inline shippable.yml:38]:
```suggestion
```
There are no 2.6 tests.

Comment (felixfontein) [reply shippable.yml]: Could you please re-add this so we can see whether it runs through?

File: shippable.yml [hunk 3]
```diff
@@ -47,6 +49,7 @@ matrix:
 
     # For Ansible 2.9, use a combination of different targets
     - env: T=2.9/sanity/1
+    - env: T=2.9/units/2.6/1
```
Comment (felixfontein) [inline shippable.yml:52]:
```suggestion
```
As mentioned above, this isn't needed.

**Comment by felixfontein** [comment]
Just to make sure you're aware of this: this PR does not affect tests for get_certificate.

**Comment by bmillemathias** [comment]
> Just to make sure you're aware of this: this PR does not affect tests for get_certificate.

Yup

File: shippable.yml [hunk 4]
```diff
@@ -34,6 +35,7 @@ matrix:
     - env: T=devel/linux/ubuntu1604/1
     - env: T=devel/linux/ubuntu1804/1
 
+    - env: T=devel/cloud/2.6/1
     - env: T=devel/cloud/2.7/1
     - env: T=devel/cloud/3.6/1
```
Comment (felixfontein) [inline shippable.yml:40]: While you're at it, why not also add:
```suggestion
    - env: T=devel/cloud/3.5/1
    - env: T=devel/cloud/3.6/1
    - env: T=devel/cloud/3.7/1
    - env: T=devel/cloud/3.8/1
    - env: T=devel/cloud/3.9/1
```

Comment (bmillemathias) [reply shippable.yml:40]: Sure, I thought they were not in for some reason.

Comment (felixfontein) [reply shippable.yml:41]: They were mostly not in because that's what ansible/ansible ran.

**Comment by felixfontein** [comment]
Aaaand it found a first problem :) There's a problem with the acme_inspect tests with Python 3.5.

File: shippable.yml [hunk 5]
```diff
@@ -34,8 +35,13 @@ matrix:
     - env: T=devel/linux/ubuntu1604/1
     - env: T=devel/linux/ubuntu1804/1
 
+    - env: T=devel/cloud/2.6/1
     - env: T=devel/cloud/2.7/1
+    - env: T=devel/cloud/3.5/1
```
Comment (felixfontein) [inline shippable.yml:40]:
```suggestion
    # - env: T=devel/cloud/3.5/1
```

**Comment by felixfontein** [comment]
I created #86 to track that failure.

**Comment by felixfontein** [comment]
Thanks a lot for doing this!

**Pull request merged**